package.json

```diff
@@ -1578,17 +1588,22 @@
 				{
 					"command": "pr.exit",
 					"when": "view == pr:github && viewItem =~ /pullrequest(:local)?:active|description/",
-					"group": "pullrequest@1"
+					"group": "1_pullrequest@1"
+				},
+				{
+					"command": "pr.pickOnVscodeDev",
+					"when": "view == pr:github && viewItem =~ /pullrequest(:local)?:nonactive|description/ && remoteName != codespaces && embedderIdentifier != github.dev",
+					"group": "1_pullrequest@2"
 				},
 				{
 					"command": "pr.refreshPullRequest",
 					"when": "view == pr:github && viewItem =~ /pullrequest|description/",
-					"group": "pullrequest@2"
+					"group": "2_pullrequest@2"
 				},
 				{
 					"command": "pr.openPullRequestOnGitHub",
 					"when": "view == pr:github && viewItem =~ /pullrequest|description/",
-					"group": "pullrequest@3"
+					"group": "3_pullrequest@3"
```

Comment:
![image](https://user-images.githubusercontent.com/38270282/223398288-bb3ec4cf-2a30-4205-9bce-8f4397a4ec07.png)


4 groups(1_, 2_, 3_, and no prefix) seems like a lot given that some onf them only have 1 item in them. Let's move `pr.openPullRequestOnGitHub` into 1_ and not have a 3_. `pr.refreshPullRequest` can just go in `pullrequest@1`